Shared/UI/Face Detection/FaceImageCaptureViewController.swift

```diff
@@ -17,7 +17,17 @@ import CoreImage.CIFilterBuiltins
 /// A live preview of the camera is shown on the main view.
 class FaceImageCaptureViewController: ViewController {
 
+    enum VideoCaptureState {
+        case idle
+        case starting
+        case capturing
+        case ending
+    }
+
+    private(set) var videoCaptureState: VideoCaptureState = .idle
+
     var didCapturePhoto: ((UIImage) -> Void)?
+    var didCaptureVideo: ((URL) -> Void)?
```

Comment:
The url to the video is returned to the handler. Currently the handler does nothing but print out the url.